### ISSUE TITLE
New updated orthographic and phonetic dictionaries to put in the plugin

### ISSUE THREAD

**Issue by Sillabix** [benoit-pierre/plover_michela#2]
Where can I send the files of the new 2.0 ortographic and phonetic dictionaries to update the plugin?

**Comment by benoit-pierre** [comment]
You can upload them here.

**Comment by Sillabix** [comment]
Ok, thanks! I can just use the same rules list we have on Eclipse in which
the prefix suffix rules are editable by the final user (very useful
function I wish Eclipse could have).

About the Michela Plugin upgrading would also be possible to put here the
2.0 dictionaries I have made, together with the link to my github page to
download the manuals?

Il giorno mar 13 nov 2018, 15:59 Benoit Pierre <notifications@github.com>
ha scritto:

**Comment by benoit-pierre** [comment]
Did you meant to reply in #3 instead?

> (very useful function I wish Eclipse could have).

You mean Plover?

**Comment by Sillabix** [comment]
Yes, sorry.
So I'll put here the new 2.0 dictionaries for the plugin upgrade. Because
is a big change (I changed the keys used to indicate the prefixes and the
suffixes for the phonetic theory and changed also the orthographic theory
accordingly) I can't specifically describe each change. So I could just
describe the general new rules I've applied to the new dictionaries.

About prefix and suffixes grammar rules they should be very simple. Just
the possibility to apply  the suffixes only if the word ending corresponds
to specific characters (es: portare + ^mi = portarmi; proponendo + ^mi
=proponendomi; contratto+ti = contratti. And for prefix just the
possibility to add in certain cases an hyphen post^ + bellico =
postbellico; post+americano= post-americano.

It would be fine also to have the possibility to manage integral prefixes/
suffixes. For instance we use to fold in a "dl"  at the beginning stroke of
a word starting with a vowel to indicate the presence of an articulate
preposition connected to the word (dell'). So if "amore" in pseudosteno is
A/MOR; DLA/MOR is "dell'amore" .




Il giorno mar 13 nov 2018, 17:56 Benoit Pierre <notifications@github.com>
ha scritto:
